Main/ClientApp/src/Components/Dialogs/ChangeMaxCollectionsDialog.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import SimpleDialog from "./SimpleDialog";
+import { Button, TextField } from "@material-ui/core";
+import { changeValue } from "../../Actions/ConfigurationActions";
+
+type ChangeMaxCollectionsDialogProps = {
+  open: boolean;
+  toggleDialogOpen: () => void;
+  maxCollections: string;
```

Comment:
why string?

Reply:
at some point i had to covert string to number, passing here string saved doing it more than once. the configuration class as you said would eliminate this problem